webpack.config.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const isProduction = process.env.NODE_ENV == 'production';
+
+const config = {
+  entry: './src/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devServer: {
+    open: true,
+    host: 'localhost',
+  },
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+    },
+  },
+  performance: {
+    hints: false,
+    maxEntrypointSize: 512000,
+    maxAssetSize: 512000,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /(node_modules|bower_components)/,
+        loader: 'babel-loader',
+        options: { presets: ['@babel/env'] },
+      },
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
```

Comment:
What do you think of usage SCSS preprocessor? It is commonly used and has a lot of outstanding things. In my opinion, if you didn't work with it earlier - this is yours best chance. Also, I would like you to use [CSS Modules](https://github.com/css-modules/css-modules) to make each *.(c|sc)ss file unique per each component to avoid styles mixing. I have left in a general comment with reference how to configure SCSS modules - take a look :)